docs/Interoperability_tests_guide/Test_case_actions.md

```diff
@@ -0,0 +1,221 @@
+ - Clear value less than – if set to true, it clears the threshold set for the monitored value. If set to false, the value is kept.
+ - Clear step – if set to true, it clears the minimum change value between two notifications. If set to false, the value is kept.
+
+### OBSERVE
+
+OBSERVE is used to initiate an observation request for changes of a specific Resource, Resources within an Object Instance or for all the Object Instances of an Object. You can define it using four attributes:
+
+![OBSERVE action configuration](images/image039.png "OBSERVE action configuration")
+
+ - Parameter name – the name of the data model parameter whose value(s) you will observe. Note that it can be specified either using the full name in the string (e.g. ‘Device.0.Manufacturer) or the numerical (e.g. 3.0.1) value of the parameter.
+ - Expected response code – if the response code you enter here equals the response code returned by the device, the test will be passed. If left blank, the server will expect the default response code defined by the LwM2M standard.
+ - Fail if already observed – if set to true, the test will fail in case there is an existing observation set on this parameter. In case there is no observation set, your observe request should be accepted and the test won’t fail. If set to false, any existing observations will be cancelled and requested again by this one and the test will be passed.
+ - LwM2M: Content format – the content format in which you send the OBSERVE request to the device. If the device doesn’t support the specified format, the test will fail.
+
+### CANCEL OBSERVE
+
+CANCEL OBSERVE is used to cancel an observation. You can define it using three attributes:
+
+![CANCEL OBSERVE action configuration](images/image040.png "CANCEL OBSERVE action configuration")
+
+ - Parameter name – the name of the data model parameter for which you will cancel an existing observation. Note that it can be specified either using the full name in the string (e.g. ‘Device.0.Manufacturer) or the numerical (e.g. 3.0.1) value of the parameter.
+ - Cancel type – the mode in which the CANCEL OBSERVE will be sent to the device.
+o	ACTIVE - CANCEL OBSERVE is sent to the device immediately.
+o	PASSIVE - CoAP RESET is sent in response to the next notification message received from the device.
+ - Expected response code – if the response code you enter here equals the response code returned by the device, the test will be passed. If left blank, the server will expect the default response code defined by the LwM2M standard.
+
+### Firmware Update
+
+Firmware Update is used to perform a firmware update operation on the test device. You can define it using five attributes:
+
+![Firmware Update action configuration](images/image041.png "Firmware Update action configuration")
+
+ - Firmware – the ID of the resource used as the firmware source.
```

Comment:
There should be a link or tip with a description of how can I upload or create a resource